pkg/kubectl/resource/builder_test.go

```diff
@@ -843,7 +846,7 @@ func TestIgnoreStreamErrors(t *testing.T) {
 	singular := false
 
 	err := b.Do().IntoSingular(&singular).Visit(test.Handle)
-	if err != nil || singular || len(test.Infos) != 2 {
+	if err == nil || singular || len(test.Infos) != 2 {
```

Comment:
@caesarxuchao Do I need rebase firstly? and then you review. I will rebase this tomorrow, or the day after tomorrow.